solutions/2221/01.py

```
@@ -0,0 +1,15 @@
+class Solution:
+    def findChampion(self, grid: List[List[int]]) -> int:
```

Ruff (0.14.8)

Undefined name `List`

(F821)